src/Pbr.cc

```diff
@@ -55,6 +55,12 @@ class sdf::PbrWorkflowPrivate
   /// \brief Emissive map
   public: std::string emissiveMap = "";
 
+  /// \brief Light map
+  public: std::string lightMap;
```

Comment:
nit: `lightMapFileName` would be clearer

Reply:
renamed. ad5f0e13d9b89f1f3ab68c8404b3a61741e56e0d